src/main/resources/views/casesListing.rocker.html

```diff
@@ -44,7 +44,17 @@
 
 <main>
     <div class="card-body">
-        <h5 class="card-title">Справи</h5>
+        <div class="card-body d-flex justify-content-between align-items-center mt-2 mb-0">
```

Comment:
Card-body is a general class for card. It's okay to use it, but it leaves more space on the bottom, than needed. Try to set needed padding on x axis and margin on y axis instead,